packages/angular-mcp-server/src/lib/tools/ds/component-contract/diff/models/schema.ts

```diff
@@ -14,7 +14,7 @@ export const diffComponentContractSchema: ToolSchemaOptions = {
       saveLocation: {
         type: 'string',
         description:
-          'Path where to save the diff result file. Supports both absolute and relative paths.',
+          'Path where to save the diff result file. Supports both absolute and relative paths. If not provided, defaults to tmp/.angular-toolkit-mcp/contracts/diffs/<component-name>-diff.json',
       },
       contractBeforePath: {
```

Comment:
Contract diff has a default saveLocation path, would be good to have an unified approach to provide same default location for contractBefore-*/afterPath